sources/core/Stride.Core/IO/NativeStream.cs

```diff
@@ -9,14 +9,17 @@ namespace Stride.Core.IO
     /// <summary>
     /// A <see cref="Stream"/> with additional methods for native read and write operations using <see cref="IntPtr"/>.
     /// </summary>
+    [Obsolete]
```

Comment:
If this abstract class is made obsolete should all classes deriving from it be obsolete as well? I'm not sure if this is right.

Reply:
That's why I asked. I will be removing it from the inheritance hierarchy shortly.